service/internal/checksum/middleware_add.go

```diff
@@ -122,10 +124,15 @@ type OutputMiddlewareOptions struct {
 	// GetValidationMode is a function to get the checksum validation
 	// mode of the output payload from the input parameters.
 	//
-	// Given the input parameter value, the function must return the validation
-	// mode and true, or false if no mode is specified.
+	// Given the input parameter value, the function must return the validation mode
 	GetValidationMode func(interface{}) (string, bool)
 
+	// Whether operation model forces middleware to validate checksum
```

Comment:
[no-op comment] - I'm totally OK w/ us talking about the model in this package, since it's `internal/` (callers can't use it).